CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(xeus_REQUIRED_VERSION 1.0.0)
 set(cppzmq_REQUIRED_VERSION 4.3.0)
 set(llvm_REQUIRED_VERSION 14)
```

Comment:
Do we need to capitalize this one, too?

Reply:
May be. If we want to parametrise build for different versions of this dependencies.